packages/app-store/_utils/getCalendar.ts

```diff
@@ -9,8 +9,8 @@ const log = logger.getChildLogger({ prefix: ["CalendarManager"] });
 export const getCalendar = (credential: CredentialPayload | null): Calendar | null => {
   if (!credential || !credential.key) return null;
   let { type: calendarType } = credential;
-  if (calendarType === "sendgrid_other_calendar") {
-    calendarType = "sendgrid";
+  if (calendarType?.endsWith("_other_calendar")) {
+    calendarType = calendarType.split("_other_calendar")[0];
```

Comment:
@hariombalhara making this as a general rule instead for specific apps as I renamed all other `_other_calendar` apps within this PR.